tools/alpine-kubectl/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.8
```

Comment:
Please update the Codeowners file + you need a job for that, right?

Reply:
This is only temporary, I am looking into moving the image to the test-infra repo